.github/workflows/test.yml

```diff
@@ -0,0 +1,59 @@
+name: test
+
+on:
+  pull_request:
+    branches:
+      - master
+
+
+jobs:
+  build-packages:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install cython --install-option='--no-cython-compile'
+      - name: Install
+        run: |
+          pip install --no-cache-dir -r requirements-dev.txt
+      - name: Lint with flake8
+        run: |
+          invoke lint
+      - name: Test import
+        run: |
+          python -m compas
+      - name: Test with pytest
+        run: |
+          invoke test
+
+  build-docs:
+    needs: build-packages
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install cython --install-option='--no-cython-compile'
+      - name: Install
+        run: |
+          pip install --no-cache-dir -r requirements-dev.txt
+      - name: Build docs with sphinx
+        run: |
+          rm -rf dist/docs && invoke docs
```

Comment:
If we decide to remove doc deployment from PRs, then we should also remove doc building, because this is built and discarded.

Reply:
The idea was to still test if the docs can be build successfully in case changes are made on the docs part in a PR. But this might not be relevant if we go ahead to align with compas_fab to deploy to different directories, right?

Reply:
exactly